Tests/Core/AWSClientRuntimeTests/AWSClientConfigurationTests.swift

```diff
@@ -63,6 +63,14 @@ class AWSClientConfigurationTests: XCTestCase {
         let subject = try await Subject(region: region, appID: appID)
         XCTAssertEqual(subject.appID, appID)
     }
+
+    // MARK: - Timeout
+
+    func test_async_configureTimeoutOptionsFromParams() throws {
```

Comment:
Do we also need a test for default timeout?

Reply:
Adding one now. Will check for 30_000 or 3_000 depending on the operating system the test is run on. There doesn't seem to be a way to force the OS in XCTest

Reply:
Removed this test as default timeout info is in CRT and not available to AWSClientConfiguration